### ISSUE TITLE
Disable prop not set to the input

### ISSUE THREAD

**Issue by LeonAlvarez** [ankurk91/vue-flatpickr-component#135]
**I'm submitting a ...**  (check one with "x")
```
[x] Bug report => search github for a similar issue or PR before submitting
[ ] Feature request
[ ] Other, please describe
```

**Tell about your platform**
* Vue.js version : 2.6.9
* This package version : 8.1.1

**Current behavior**
Disabled property not being set to the input, im passing true to it.
![image](https://user-images.githubusercontent.com/22636643/54762630-bd354880-4bf4-11e9-9d5d-82f6520b3877.png)
But its not applied to the input
![image](https://user-images.githubusercontent.com/22636643/54762667-d4743600-4bf4-11e9-85bf-3219fe1bd030.png)

**Expected behavior**
Disabled prop should disable the input, I take a look and it should be fixed on this version. Any idea of waht might be causing this issue ?

**Comment by ankurk91** [comment]
Please share your config or a js fiddle that can reproduce this issue.
Screenshot was not helpful

**Comment by LeonAlvarez** [comment]
https://jsfiddle.net/xe5hLpkw/

**Comment by doncatnip** [comment]
It's the same with no config at all.

**Comment by ankurk91** [comment]
Should be fixed in v8.1.2, thanks for reporting.
